packages/shared/src/graphql/users.ts

```diff
@@ -181,6 +181,7 @@ const READING_HISTORY_FRAGMENT = gql`
       numUpvotes
       bookmarked
       numComments
+      private
```

Comment:
This is not great, it should still show in your actual reading history, just not the ones that are used for sharing I think.

Reply:
We already have the condition in the ShareModal to not show private posts. In reading history, the context menu also have that logic.

Reply:
Sorry my bad, I was thinking you introduced "isPublic" 😅